activesupport/CHANGELOG.md

```diff
@@ -205,5 +212,4 @@
 
     *Jordan Thomas*
 
-
```

Comment:
This line should be here.

Reply:
I have added back this line which was accidentally removed.